WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShipping Customs/WooShippingCustomsFormViewModel.swift

```diff
@@ -55,31 +62,27 @@ final class WooShippingCustomsFormViewModel: ObservableObject {
         listenToItemsRequiredInformationValues()
         listenForRequiredInformation()
         listenForInternationalTransactionNumberIsRequired()
+        listenForRequiredInformationCompletedUponPreFill()
     }
 
-    @Published private(set) var itemsViewModels: [WooShippingCustomsItemViewModel] = []
+    /// WOOMOB-734
+    /// Solves the issue where a pre-filled form becomes complete without a manual submission
+    ///
+    /// Listens for the `requiredInformationIsEntered` state
+    /// As soon as all required info is entered, calls the `emitForm` just once
+    func listenForRequiredInformationCompletedUponPreFill() {
+        $requiredInformationIsEntered
+            .first { $0 == true }
```

Comment:
Using `first` here can be risky. I still see the purchase failure with the following steps (edge case):

- Select an order with a US address as the destination.
- Edit the address to outside of the US.
- Proceed to purchase -> same error about missing customs form.